app/controllers/decks_controller.rb

```diff
@@ -0,0 +1,61 @@
+class DecksController < ApplicationController
+  def index
+    @decks = Deck.all
+  end
+
+  def new
+    @deck = Deck.new
+  end
+
+  def create
+    @deck = Deck.new(deck_params.merge(user: current_user))
+
+    if @deck.save
+      redirect_to decks_path
+    else
+      render 'new'
+    end
+
+  end
+
+  def edit
+    @deck = Deck.find(params[:id])
+  end
+
+  def update
+    @deck = Deck.find(params[:id])
+
+    if @deck.update(deck_params)
+      redirect_to decks_path
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.